docs/monitoring/autodiscovery.md

```diff
@@ -0,0 +1,85 @@
+title: Sematext Autodiscovery for Monitoring
+description: Sematext Autodiscovery Monitoring automatically scans for services that can be monitored using Sematext Agent and, depending on your configuration, automatically sets up monitoring agents. 
+
+## Autodiscovery Monitoring
+
+The centerpiece of Autodiscovery feature is Sematext Agent which scans for services that can be monitored by Sematext supported integrations. It works in both bare-metal/virtual machine and in container environments. Data about discovered services is displayed under Discovery.
```

Comment:
Link "supported integrations" to jump to the Q below?
Link "Sematext Agent" to https://sematext.com/docs/agents/sematext-agent/